dom0/sd-svs-disp-files.sls

```diff
@@ -19,3 +19,18 @@ sd-svs-disp-install-mimetype-handler-package:
       - securedrop-workstation-svs-disp
     - require:
       - sls: fpf-apt-test-repo
+
+paxctld:
+  service.running:
+    - enable: True
+    - reload: True
```

Comment:
The task should have a more explicit name, as all tasks exist in the same namespace once the various states have been merged. Compare with the task below: `sd-svs-disp-install-libreoffice`, rather than `libreoffice`.

More relevant for predictable service state is that we need to inform Qubes (via dom0) that a given VM should start a service. See description in #231, specifically the mention of `qvm-service`. Following that approach will not only resolve concerns about the libreoffice install in this PR, but also allow us to resolve #231, since we'll be managed the service on all the requisite VMs, not just one.